File: Lecture 004 Recursion Phod Denge/Fib.py
def fib(n):
    #Base Case
    if n<=1:
        return n
    #Bharosa
    prev= fib(n-1)
    pprev= fib(n-2)

    #tumhara Kaam 
    return prev+ pprev
  
    
n= int(input())
ans= fib(n)
print(n,"th fibonacci is:",ans)

File: Lecture 004 Recursion Phod Denge/Fib.java
import java.util.Scanner;

public class Main {
	
	public static int fib(int n) {
         // base case 
		if(n<=1) {
			return n;
		}

       //Bharosa 
       int prev= fib(n-1);
       int pprev= fib(n-2);

       //Tumhara kaam

       return prev + pprev;
	}
	
	public static void main(String[] args) {
		Scanner sc = new Scanner(System.in);
		int n = sc.nextInt();
		int ans = fib(n);
	    System.out.println(n + "th fibonacci is : "+ans);
	}
}